run.sh

```diff
@@ -38,18 +76,24 @@ if [[ "x$1" != "x--restart" ]]; then
   fi
 fi
 
-pushd .
-  # Ensure rustc is updated
-  echo "* Updating rust if necessary..."
+pushd . &> /dev/null
   cd runtime
-  multirust override nightly-2015-08-10
 
-  # Compile runtime server
-  echo "* Compiling server... (This takes a while)"
+  # Ensure rustc is updated.
+  multirust list-toolchains | grep "$rustVersion" &> /dev/null
```

Comment:
If we detect that rust doesn't have `$rustVersion` installed, we know `multirust override "$rustVersion"` will trigger an installation. Since we echo "* Updating rust..." when this happens, and since multirust is blocking, it would be nice to printf that message and echo "done." when it completes the way you have for the other build phases.